CondFormats/L1TObjects/interface/CaloConfig.h

```diff
@@ -0,0 +1,44 @@
+///
+/// \class l1t::CaloConfig
+///
+/// Description: Placeholder for calorimeter trigger runtime configuration
+///
+/// Implementation:
+///
+///
+
+#ifndef CaloConfig_h
+#define CaloConfig_h
+
+#include <memory>
+#include <iostream>
+#include <vector>
+#include <string>
+#include <cmath>
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "CondFormats/L1TObjects/interface/LUT.h"
+
+namespace l1t {
+
+
+  class CaloConfig {
+
+  public:
+
+    enum { Version = 1 };
+
+    CaloConfig() { version_=Version; }
```

Comment:
@mulhearn here I can see an implicit conversion from non-strongly typed `enum` to `unsigned int`. Can this be made explicit?